developer-docs-site/docs/tutorials/validator-node/using-source-code.md

```diff
@@ -44,31 +44,31 @@ With your development environment ready, now you can start to setup your Validat
 7. Generate key pairs (node owner key, consensus key and networking key) in your working directory.
 
     ```
-    cargo run -p aptos -- genesis generate-keys --output-dir ~/$WORKSPACE
+    cargo run --release -p aptos -- genesis generate-keys --output-dir ~/$WORKSPACE
```

Comment:
I think if `--release` is omitted it builds a debug binary, which is slower but still a valid one.

Reply:
See the NOTE below step 9 here in this doc:https://aptos.dev/tutorials/full-node/run-a-fullnode/#using-aptos-core-source-code

Reply:
Yes, it is true, my point here was that the final build for the node actually done in release. But having both release and debug builds is a waste of time and disk absolutely for nothing